src/ansi.rs

```diff
@@ -678,6 +686,50 @@ bitflags! {
     }
 }
 
+/// This only applies to keys corresponding to ascii characters.
```

Comment:
Usually the first line is a headline on what the struct is actually about.

So maybe make this the first paragraph and add a first line like this:

```
/// XTMODKEYS modifyOtherKeys state.
```